languages/php/INSTALL.md

```diff
@@ -0,0 +1,58 @@
+# PHP Installation
+
+## Introduction
+
+Composer is used to build PHP Bitwarden client library.
+
+## Prerequisites
+
+- PHP >= 8.0
+- FFI extension enabled in PHP configuration
+- Composer
+- Bitwarden SDK native library, which is expected in one of below locations, depending on the OS and architecture.
+  If you prefer to build SDK yourself, see [SDK README.md](../../README.md) for instructions.
+    - Windows:
+      - x86_64: `C:\lib\windows-x64\bitwarden_c.dll`.
+      - The drive letter is determined by the workdir of the PHP process, usually `C:`
+    - Linux:
+      - x86_64: `/lib/linux-x64/libbitwarden_c.so`
+    - macOS:
+      - x86_64: `./src/lib/macos-x64/libbitwarden_c.dylib`
+      - aarch64: `./src/lib/macos-arm64/libbitwarden_c.dylib`
+      - Where the `./src` is relative path to the [src](./src) directory.
```

Comment:
Based on what the workflow for this used to look like, https://github.com/bitwarden/sdk/blob/8ea2f79c6bdddbfb1a6fa051d0461a1a9fb724c5/.github/workflows/publish-php.yml#L59-L71. 
I'm assuming the intent was to have everything be `./src/lib/<os>/<libbitwarden>`.

My suggestion would be to update `languages/php/src/BitwardenLib.php` and have each OS target have similar behavior.